app/app.tsx

```diff
@@ -230,30 +230,33 @@ export const App = (): JSX.Element => {
     return null
   }
 
+  // TODO: Fix this. It does not work currently
```

Comment:
Could you add more context to the TODO? If I saw this I wouldn't know what isn't working (is it the type?). Or even better link it to an issue in GitHub.